src/organisms/cards/StackedCard/__tests__/stacked_card.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+// import sinon from 'sinon';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import StackedCard from '../index';
+
+describe('StackedCard', () => {
+  // let defaultProps;
+  //
+  // beforeEach(() => {
+  //   defaultProps = {
+  //     // Add default props here
+  //   };
+  // });
+
+  it('is a <StackedCard>', () => {
+    const wrapper = shallow(<StackedCard />);
+
+    expect(wrapper).to.be.ok;
```

Comment:
are you just testing here that the component renders something? if that's the case I think
```
expect(wrapper.length).to.equal(1);
```
would be a better assertion because wrapper is always truthy, even for components that render falsey values